tests/acceptance/conftest.py

```diff
@@ -195,6 +200,20 @@ def request(
             merged_headers = deepcopy(self.default_headers)
             if isinstance(headers, dict):
                 merged_headers.update(headers)
+            if (headers is None or "Authorization" not in headers.keys()) and stack[
+                "AuthMethod"
+            ] == "IAM":
+                creds = boto3.Session().get_credentials().get_frozen_credentials()
+                if "json" in kwargs:
+                    request = AWSRequest(
+                        method=method.upper(), url=url, data=json.dumps(kwargs["json"])
+                    )
+                else:
+                    request = AWSRequest(method=method.upper(), url=url)
+                SigV4Auth(
+                    creds, "execute-api", getenv("AWS_DEFAULT_REGION", "eu-west-1")
```

Comment:
do we default to eu-west-1 anywhere else?

Reply:
Yes, as you guessed I was using this default to be consistant, eu-west-1 is default option when we create the temp bucket.  This is found later in the same file (line 569).